doc/source/comparison_with_r.rst

```diff
@@ -530,6 +530,103 @@ For more details and examples see :ref:`categorical introduction <categorical>`
 :ref:`differences to R's factor <categorical.rfactor>`.
 
 
+String Processing
+-----------------
+
+Length
+~~~~~~
+
+R determines the length of a character string with the ``nchar`` function. 
+``nchar`` includes leading and trailing blanks.  Use ``nchar`` and ``trimws`` 
+to exclude leading and trailing blanks. 
+
+.. code-block:: none
```

Comment:
http://pygments.org/docs/lexers/#lexers-for-the-r-s-languages

`r` or `rconsole` should work.